src/main/java/pl/edu/pwr/kpz/lostanimalsbackend/model/dto/AnimalRequestDTO.java

```diff
@@ -1,15 +1,23 @@
 package pl.edu.pwr.kpz.lostanimalsbackend.model.dto;
 
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.NotNull;
 import lombok.Data;
 import pl.edu.pwr.kpz.lostanimalsbackend.model.entities.Sex;
 
 @Data
 public class AnimalRequestDTO {
+    @NotBlank(message = "Animal name shouldn't be blank")
     private String name;
     private String chip;
+    @NotNull(message = "Animal sex shouldn't be null")
     private Sex sex;
 
+    @Min(value = 1, message = "Wrong owner id in animal")
```

Comment:
Wydaje mi się, że trzeba zmienić opis typu "Wrong ...." na bardziej dokładną wiadomość, w stylu: "Owner id must be greater than ...!"

Reply:
to chyba będzie trzeba to zmienić u wszystkich